ci-operator/config/openshift/cluster-api/openshift-cluster-api-master.yaml

```diff
@@ -45,7 +45,7 @@ resources:
 test_binary_build_commands: make test
 tests:
 - as: unit
-  commands: make test
```

Comment:
Why do we want to refuse using make here?

Reply:
For this case, it's easier than patching upstream one